app/src/components/pages/ScheduleTimetable.tsx

```diff
@@ -4,22 +4,25 @@ import 'devextreme/dist/css/dx.light.css';
 import Appointment from 'components/organisms/Appointment';
 import classData from 'data/clean.json';
 import { parseCalendarCourse, parseCalendarTeacher } from 'utils/utils';
-// import { ICourse } from 'components/shared/interfaces/timetable.interfaces';
-// import { useLocation } from 'react-router-dom';
+import { Location, useLocation } from 'react-router-dom';

```

Comment:
If it isn't too much work can we add a filter indicator somewhere so Rich knows how the calendar is currently filtered? I dont think we need UI for now but maybe just the querystring or route param

Reply:
<img width="748" alt="Screen Shot 2022-07-07 at 5 33 52 PM" src="https://user-images.githubusercontent.com/35615145/177893312-ef0de941-17cc-4c26-8cc5-77599fbcd531.png">